src/tools/rustdoc.md

```diff
@@ -0,0 +1,8 @@
+# `rustdoc`
+
+`rustdoc` has many options and can accept a wide variety of arguments,
+and its behavior can vary depending on the values of several environment
+variables.
+
+We document its command-line options, arguments, and operative environment
```

Comment:
You can just link to `../rustdoc/index.html` for documenting rustdoc (and `../cargo/index.html` for cargo)

Reply:
oh dang it, I didn't even know about those. I'll link to the references.